frontend/simple/js/backend/hapi.js

```diff
@@ -85,17 +85,21 @@ export class HapiBackend extends Backend {
     if (this.subscriptions().indexOf(contractId) === -1) {
       return console.error('HapiBackend.unsubscribe: not subscribed!', contractId)
     }
-    var res = await primus.unsub(contractId)
-    store.commit('removeContract', contractId)
+    let res = await primus.unsub(contractId)
```

Comment:
changing a `var` to a `let` doesn't do much here, but while we're on the subject, you might as well chat it to a `const` since that might have some (probably imperceptible) performance improvement (that might add up if we're consistent about it elsewhere).